docs/commands.md

```diff
@@ -21,6 +21,8 @@ Optional args:
    Adding this flag will keep data in an uncompressed form. Compression not only saves
    memory but usually improve performance due to lower number of memory accesses. 
  * CHUNK_SIZE - amount of memory, in bytes, allocated for data. Default: 4000.
+ * DUPLICATE_POLICY - configure what to do on duplicate sample, possible values: BLOCK, FIRST, LAST.
```

Comment:
So we didn't choose `ON_DUPLICATE`?

Reply:
@K-Jo fixed that, in TS.ADD it will be `ON_DUPLICATE`

Reply:
my bad, fixed it.